jsonrpc/helper.go

```diff
@@ -74,7 +80,13 @@ type headerGetter interface {
 // GetBlockHeader returns a header using the provided number
 func GetBlockHeader(number BlockNumber, store headerGetter) (*types.Header, error) {
 	switch number {
-	case PendingBlockNumber, LatestBlockNumber:
+	case SafeBlockNumber:
+		fallthrough
+	case FinalizedBlockNumber:
+		fallthrough
+	case PendingBlockNumber:
```

Comment:
Maybe to switch order of Pending and Latest?

Reply:
Fixed https://github.com/Ethernal-Tech/blade/pull/412/commits/236588579802581beb09af17978f579548fd496a